### DISCUSSION TITLE
Send notifications when a user successfully or unsuccessfully login

### DISCUSSION THREAD

**Discussion by KasaiJo** [authelia/authelia#7412] [Q&A]
Hi all,

I wonder if I've missed something. Is it possible to send notifications of connection events (failure, success, etc.)?
I'd like to use Gotify in combination with authelia.

**Comment by glime-ai** [comment]
Please note this response was created using generative AI leveraging previously available information from the repository. This information may be helpful however it is our strong recommendation that you carefully consider the information before blindly applying it.

---
Sending notifications via gotify is not currently supported.

~~Yes, it is possible to send notifications of connection events such as failures and successes using Authelia. Authelia supports various notification providers, including SMTP and filesystem. However, for using Gotify, you would need to configure it as a custom notification provider.~~

To set up Gotify with Authelia, you would need to follow these general steps:

1. **Configure Gotify Server**: Ensure your Gotify server is up and running.
2. **Authelia Configuration**: Modify the Authelia configuration to include Gotify as a notification provider. This typically involves specifying the Gotify server URL and the necessary authentication tokens.

Here is a basic example of how you might configure Gotify in Authelia's configuration file:

```yaml
notifier:
  gotify:
    url: "https://gotify.example.com"
    token: "YOUR_GOTIFY_APP_TOKEN"
```

3. **Event Triggers**: Ensure that the events you want to be notified about (e.g., login failures, successes) are configured to trigger notifications.

For more detailed instructions, you may need to refer to the [Authelia documentation](https://www.authelia.com/configuration/notifications/introduction/) and the [Gotify API documentation](https://gotify.net/docs).

This comment was generated by [Glime](https://glimelab.ai/?utm_source=6fsHkcyufjQLaB&utm_medium=github).

**Reply by AndreiArdelean1** [reply]
Does Authelia support gotify for notifications? I see no references to it in the docs. I see only filesystem and smtp

**Reply by james-d-elliott** [reply]
No it doesn't.
